content/en/tracing/setup/nodejs.md

```diff
@@ -94,7 +101,9 @@ For details about how to how to toggle and configure plugins, check out the [API
 
 | Module           | Versions | Support Type    | Notes                                      |
 |------------------|----------|-----------------|--------------------------------------------|
+| [connect][45]    | `>=2`    | Fully Supported |                                            |
 | [express][10]    | `>=4`    | Fully Supported | Supports Sails, Loopback, and [more][11]   |
+| [fastify][47]    | `>=1`    | Fully Supported |                                            |
```

Comment:
```suggestion
| [fastify][47]    | `>=1`    | Fully Supported |Requires `dd-trace-js` v0.13.0+                      |
```

Reply:
As discussed, let's leave this out for now since it would be difficult to track the exact library version support by tracer version.